src/middleware/createInterview.js

```diff
@@ -0,0 +1,30 @@
+import {CREATE_INTERVIEW} from "../actions/interviewActions";
+import {constants} from "../utility/constants";
+
+const createInterview = store => next => async action => {
+    next(action);
+
+    if (action.type !== CREATE_INTERVIEW) {
+        return;
+    }
+
+    const dispatch = store.dispatch;
+
+    try {
+        console.log(action.interview)
+        const data = await fetch(constants.host + "/interviews/", {
+            headers: {
+                "Content-Type": "application/json"
+            },
+            method: "POST",
+            body: action.interview,
+
+        });
```

Comment:
you didn't handle the promise for this call why? how will you notify the user that interview is created